src/obb_rules/serializer/reader.cljc

```diff
@@ -97,14 +101,18 @@
       [deploy1 deploy2]
       [deploy2 deploy1])))
 
+(defn- add-from-deploy-to-stash
+  "Gets the quantity of a unit deployed on a raw deploy action, and
+  adds it to the given stash map"
+  [stash raw-unit]
```

Comment:
I guess this was meant to be called `raw-action` instead of `raw-unit`. ;)

This is a detail and maybe not worth changing at all but this function (and the associated changes) probably belonged to the next commit.